dcos/rpcclient.py

```diff
@@ -145,7 +144,8 @@ def http_req(self, method_fn, path, *args, **kwargs):
                 request_method=e.response.request.method,
                 request_url=e.response.request.url,
                 json_body=json_body)
-            raise DCOSException(message)
+
+            raise DCOSException(message) from e
```

Comment:
Customer first here, we can't show a stack trace. Why can't you use the debug flag and then see the stack trace there? You are welcome to add more logging wherever else it may be helpful.